velero-plugins/imagestream/backup.go

```diff
@@ -206,3 +215,66 @@ func (p *BackupPlugin) coreClient() (*corev1.CoreV1Client, error) {
 	}
 	return client, nil
 }
+
+func copyImage(p *BackupPlugin, src, dest string) (string, error) {
```

Comment:
Do you want this function exported? Or will the reuse of this be in the same package?

Reply:
For now, it should only be needed within the package.